docs/pages/hdmi_cec_source_tests.md

```diff
@@ -0,0 +1,176 @@
+# HDMI CEC Test Document
+
+## Version History
+
+| Date(DD/MM/YY) | Comment | Version |
+| -------------- | ------- | ------- |
+| 13/02/24 | First Release | 1.0.0 |
+
+## Table of Contents
+
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [Overview](#overview)
+- [HDMI-CEC RDK HAL Functionality](#hdmi-cec-rdk-hal-functionality)
+
+## Acronyms, Terms and Abbreviations
+
+- `CEC` - Consumer Electronics Control
+- `HAL` - Hardware Abstraction layer
+- `HDMI` - High Definition Multimedia Interface
+- `L2` - Level 2 Testing ()
+- `L3` - Level 3 Testing ()
+- `Sink device` - Any piece of equipment or technology that provides an input signal or data to another device or system.
+- `Source Device` - Any piece of equipment or technology that receives an input signal or data from another device or system.
+
+## Scope
+
+This document defines the requirements for testing the HDMI CEC device from a level 2/3 based on being a source device.
+
+## Overview
+
+Consumer Electronics Control (CEC) is a single-wire bidirectional bus within an HDMI system, facilitating communication among interconnected products. HDMI-CEC establishes a protocol enabling high-level control functions between audiovisual devices linked via an HDMI network, facilitating communication and control among them. Communication can occur in either Direct messaging mode or Broadcast mode. 
+
+## HDMI-CEC RDK HAL Functionality
+
+The HAL layers within RDK serve as a bridge between the underlying low-level SoC drivers and the higher-level RDK layers that utilize the functionality offered by these HAL functions. Specifically concerning the CEC Module, the HAL layers facilitate the following functionalities:
+
```

Comment:
Logical Address discovery is also part of CEC activity on source devices.

Reply:
Updated as requested